src/main/java/com/hubspot/jinjava/interpret/LazyExpression.java

```diff
@@ -6,20 +6,35 @@
 public class LazyExpression implements Supplier {
   private final Supplier supplier;
   private final String image;
+  private final MEMOIZATION memoization;
   private Object jsonValue = null;
 
-  private LazyExpression(Supplier supplier, String image) {
+  public enum MEMOIZATION {
```

Comment:
```suggestion
  public enum Memoization {
```

Sorry, got too capitalization-happy with my suggestion earlier.

Reply:
I _trusted_ you!!! Fixed in ecfb2529